src/containers/DataDisplay/DataViewTable.tsx

```diff
@@ -171,15 +172,24 @@ const DataViewTableContent: React.FC<DataViewTableContentProps> = ({
           </Table.Row>
         </Table.Header>
         <Table.Body>
-          {tableRows.map(({ id, rowValues }: { id: number; rowValues: any }) => (
+          {tableRows.map(({ id, rowValues }) => (
             <Table.Row
               key={`row_${id}`}
               className="clickable"
               onClick={() => showDetailsForRow(id)}
             >
               {rowValues.map((value: any, index: number) => (
                 <Table.Cell key={`value_${index}`}>
-                  {getCellComponent(value, headerRow[index], id)}
+                  <TableCellMobileLabelWrapper
+                    label={headerRow[index].title}
+                    // These values are not currently dynamic, hence no rowData
+                    // sent
+                    hideLabel={headerRow[index].formatting?.hideLabelOnMobile}
+                    hideCell={headerRow[index].formatting?.hideCellOnMobile}
```

Comment:
These values added in Data View configuration in database. Have added additional note to back-end docs.